<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubForm_CRC</class>
 <widget class="QWidget" name="SubForm_CRC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CRC校验设置</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="check_box_crc_enable">
       <property name="text">
        <string>校验使能</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <widget class="QLabel" name="label_crc_range">
        <property name="text">
         <string>检验区间</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_crc_type">
        <property name="text">
         <string>校验类型</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <widget class="QComboBox" name="combo_box_crc_range"/>
       <widget class="QComboBox" name="combo_box_crc_type"/>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <widget class="QLabel" name="label_crc_width">
        <property name="text">
         <string>宽度(WIDTH)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_crc_poly">
        <property name="text">
         <string>多项式(POLY)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_crc_formula">
        <property name="text">
         <string>多项式公式</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_crc_init">
        <property name="text">
         <string>初始值(INIT)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_crc_xorout">
        <property name="text">
         <string>结果异或值(XOROUT)</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSplitter" name="splitter_4">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <widget class="QComboBox" name="combo_box_crc_width"/>
       <widget class="QLineEdit" name="line_edit_crc_poly">
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_edit_crc_formula">
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_edit_crc_init">
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_edit_crc_xorout">
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="check_box_in_reversal">
       <property name="text">
        <string>输入反转</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="check_box_out_reversal">
       <property name="text">
        <string>输出反转</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSplitter" name="splitter_5">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="button_crc_confirm">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_crc_cancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="check_box_high_first">
       <property name="text">
        <string>高字节在前</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_byte_order">
       <property name="text">
        <string>字节顺序</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
